CMakeLists.txt

```diff
@@ -94,3 +98,37 @@ if(QHOTKEY_INSTALL)
 
     export(TARGETS qhotkey FILE QHotkeyConfig.cmake)
 endif()
+
+if(UNIX)
+    # General settings
+    set(CPACK_GENERATOR "TBZ2")
+    set(CPACK_PACKAGE_NAME "libqhotkey")
+    set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Global hotkey library for Qt software")
+    set(LICENSE_FILE "LICENSE")
+    set(README_FILE "README.md")
+    set(PROJECT_URL "https://skycoder42.github.io/QHotkey/")
+    set(COMPRESION_TYPE "xz")
+    set(CPACK_PACKAGE_VENDOR "Javier O. Cordero Pérez")
+    set(CPACK_PACKAGE_CONTACT "javiercorderoperez@gmail.com")
+    set(CPACK_PACKAGE_VERSION_MAJOR "${RELEASE_VERSION_MAJOR}")
+    set(CPACK_PACKAGE_VERSION_MINOR "${RELEASE_VERSION_MINOR}")
+    set(CPACK_PACKAGE_VERSION_PATCH "${RELEASE_VERSION_MICRO}")
+    set(CPACK_PACKAGE_VERSION "${RELEASE_VERSION}")
+    set(CPACK_RESOURCE_FILE_README "${CMAKE_SOURCE_DIR}/${README_FILE}")
+    # CPACK: DEB Specific Settings
+    set(CPACK_DEBIAN_PACKAGE_SECTION "Libraries")
+    set(CPACK_DEBIAN_PACKAGE_HOMEPAGE ${PROJECT_URL})
```

Comment:
You can set the description in the [project()](https://cmake.org/cmake/help/latest/command/project.html) and it will be propagated.